migration/src/m20230726_013107_remove_not_null_constraint_from_group_value.rs

```diff
@@ -0,0 +1,32 @@
+use digital_asset_types::dao::asset_grouping;
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                sea_query::Table::alter()
+                    .table(asset_grouping::Entity)
+                    .modify_column(ColumnDef::new(asset_grouping::Column::GroupValue).null())
+                    .to_owned(),
+            )
+            .await?;
+        Ok(())
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
```

Comment:
_**Something to consider:**_
If the column already has `NULL` values, the rollback won't work. For that, we'll need to remove/modify the `NULL` values first. Any suggestions here would be helpful.

Reply:
Add a comment explaining this and that's ok